classes/Tools.php

```diff
@@ -688,7 +688,7 @@ public static function setCurrency($cookie)
     public static function getCldr(Context $context = null, $language_code = null)
     {
         static $cldr_cache;
-        if ($context) {
+        if ($context && $context->language instanceof Language) {
```

Comment:
Having a Context doesn't ensure to have a language class here, most of the time a notice is logged but our test suite convert it to an exception.